src/box/box.scss

```
@@ -144,7 +144,7 @@
 
     // Row dragging styles
     &.sortable-chosen {
-      background-color: var(--color-bg-primary-light);
+      background-color: var(--color-bg-primary);
```

I think this used to be `$bg-gray-light` which is `$gray-000`. So to not cause regressions we could use `--color-bg-tertiary` that is also `$gray-000`.

```suggestion
      background-color: var(--color-bg-tertiary);
```